hack/observability/kube-state-metrics/metrics/regen.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Would be great to have a generator replacing this script and yaml files.
+
+set -e
+
+DIR="$(dirname ${BASH_SOURCE[0]})"
```

Comment:
Should we call this script in some gen + verify-gen target?

Reply:
Sounds better 👍 

I moved the script to a make target and removed it and added it to the `generate` target. This way it is also included in `verify-generate`.

Reply:
Thx!